SynthDet/Assets/Scripts/Randomizers/CameraPostProcessingMetricReporter.cs

```diff
@@ -0,0 +1,71 @@
+﻿using System;
+using System.Collections.Generic;
+using SynthDet.RandomizerTags;
+using UnityEngine;
+using UnityEngine.Perception.GroundTruth;
+using UnityEngine.Perception.Randomization.Randomizers;
+using UnityEngine.Rendering;
+using UnityEngine.Rendering.Universal;
+
+namespace SynthDet.Randomizers
+{
+    /// <summary>
+    /// Reports post processing metrics for object that contain a post-processing volume component and are tagged with <see cref="CameraPostProcessingMetricReporter"/>. 
+    /// </summary>
+    [Serializable]
+    [AddRandomizerMenu("SynthDet/My Camera Post-processing Metric Reporter")]
+    public class CameraPostProcessingMetricReporter : Randomizer
```

Comment:
Similar comment as elsewhere - why not include the metrics in the camera randomizer?

Reply:
-resolved offline
Summary: The Randomizer workflow provides a powerful way of coordinating timing of code execution within the Scenario, and thus makes it easy to make sure the reporting is done after all other Randomizers are done. Thus, for the time being, we will keep this pattern for metric reporters. This may be iterated on in future versions.